src/apis/impl/AuthenticationApiImpl.ts

```diff
@@ -217,7 +234,7 @@ export class AuthenticationApiImpl implements AuthenticationApi {
 
     const dataOwner = await authenticatedApi.baseApi.cryptoApi.getDataOwner(dataOwnerId);
     if (dataOwner == null) {
-      throw Error("Your user is not a data owner");
+      throw this.errorHandler.createErrorWithMessage("Your user is not a data owner");
```

Comment:
`iCure could not retrieve information of user ${loggedUser.id} data owner ${dataOwnerId}. Make sure this data owner (either a patient, a healthcare party or a device) really exists`